filestore/s3_test.go

```diff
@@ -49,6 +52,15 @@ func (t *TestSuite) SetupSuite() {
 	t.minio = minio
 	t.deleteTempDir = tcfg.DeleteTempDir
 	t.loggerhook = logrust.NewGlobal()
+	
+	customTransport := &http.Transport{
+	    TLSClientConfig: &tls.Config{InsecureSkipVerify: false},
+        }
+	customHTTPClient := &http.Client{
+                Transport:        customTransport,
+		Timeout:         24 * time.Hour }
```

Comment:
I still need to make this a variable shared with the server in app/blobstore.go.

Reply:
wrong file for this comment